src/vue/modules/document-upload-form/index.vue

```diff
@@ -0,0 +1,200 @@
+<template>
+  <div class="document-upload-form">
+    <form
+      class="app__form"
+      @submit.prevent="isFormValid() && showConfirmation()"
+    >
+      <div class="app__form-row">
+        <div class="app__form-field">
+          <file-field
+            :label="'document-upload-form.document-lbl' | globalize"
+            :note="'document-upload-form.file-type-note' | globalize"
+            accept="image/*, .pdf"
+            :document-type="DOCUMENT_TYPES.identity_card"
```

Comment:
general_private?

Reply:
`general_private` is a policy, not the type. The policy depends on a document type here. Should we declare new document type for healthcare documents?

Reply:
@ardier16 +

Reply:
Added